apps/src/applab/designMode.js

```diff
@@ -180,7 +192,9 @@ designMode.updateProperty = function(element, name, value) {
     case 'style-width':
       var newWidth = appendPx(value);
       element.style.width = newWidth;
-      element.parentNode.style.width = newWidth;
+      if (isDraggableContainer(element.parentNode)) {
+        element.parentNode.style.width = newWidth;
+      }
```

Comment:
Previously we would only ever edit properties while in design mode, at which point they are always wrapped. Now we sometimes edit properties via API, at which point in time the element is not wrapped and we don't actually want to be mucking with element.parentNode.